sdks/python/apache_beam/testing/load_tests/load_test_metrics_utils.py

```diff
@@ -111,11 +112,15 @@ def _prepare_all_metrics(self, metrics):
       insert_rows.append(counter_dict)
 
     dists = metrics['distributions']
-    if len(dists) > 0:
-      runtime = RuntimeMetric(dists, submit_timestamp, metric_id)\
-        .as_dict()
-      insert_rows.append(runtime)
-
+    sorted_dists = sorted(dists, key=lambda x: x.key.metric.name)
```

Comment:
re line 110: let's use brackets for implicit line continuation and remove '\'. From PEP-0008:
> The preferred way of wrapping long lines is by using Python's implied line continuation inside parentheses, brackets and braces

Reply:
That's right. I changes with all similar cases.